Makefile

```diff
@@ -28,17 +35,17 @@ $(BUILD_DIR)/%.o: %.c
 	@mkdir -p $(BUILD_DIR)
 	$(CC) $(CFLAGS) $(COVERAGE_CFLAGS) -I. -c $< -o $@
 
-.PHONY = clean
+.PHONY: clean
```

Comment:
**praise**: Ah! I would always miss this and think that Makefile isn't working.

**question**: Should we use some other simpler (_and idiot-proof_) build system? Like Scons, etc.?

Reply:
> question: Should we use some other simpler (and idiot-proof) build system? Like Scons, etc.?

I think we should shift to SCons if this MakeFile is going to be complex in future or if we need a simpler build system. However, SCons automatically determines dependencies and recompiles only the necessary parts when source files change. Which sure is a plus-point.

What do _you_ think?

Reply:
After, the project has progressed some more, we can move it to another build system.